src/odemis/acq/stream/_sync.py

```diff
@@ -143,6 +144,13 @@ def __init__(self, name, streams):
                 self._sstage = s._sstage
                 break
 
+        # Check for an EBIC detector with its own scan box
+        for s in streams:
+            if hasattr(s, "_ebic") and s._ebic.role == "ebic-detector":
+                self.useEBIC = True  # bool for the presence of the EBIC
+                self._ebic = s._ebic  # the actual EBIC component
+                break
```

Comment:
We shouldn't have any special code for the "EBIC". I think it should just check whether the detector has also a scanner info (ie, .resolution and .dwellTime). If so, they should be set properly.